### ISSUE TITLE
Unable to run script in xcode

### ISSUE THREAD

**Issue by nomimalik1769** [applanga/sdk-ios#6]
I am trying to run script for auto update applanga setting file but I am facing this error.
I am using this script. `bash "$SOURCE_ROOT/Pods/Applanga/Applanga.framework/update-settingsfile.sh" "$SOURCE_ROOT/$TARGET_NAME"
`

![image](https://user-images.githubusercontent.com/26509387/45251655-2c3b0600-b363-11e8-8e18-8eb272489ba6.png)

**Comment by Neirys** [comment]
Any news on that ?

**Comment by steffen-roemer** [comment]
this has since then been fixed

**Comment by btrane** [comment]
I'm trying to integrate the SDK for the first time, but I'm seeing this same error when adding this script to my build phases or running it via CLI. I'm currently using the `2.0.129` release via Cocoapods.

**Comment by applanga-rich** [comment]
Hi @btrane are you able to share the build log with me so that we can work out what's going wrong?

I just setup a test project and it's working fine with the following log out put

![image](https://user-images.githubusercontent.com/53563197/71004766-ac155b80-20e2-11ea-8175-646b048ba0cb.png)

**Comment by btrane** [comment]
Thanks for the quick response! Here's my log output from my current project:
```
Applanga SDK Path: /Users/btrane/myproject/Pods/Applanga/Applanga.framework
2019-12-16 14:55:09.810 defaults[19579:22300818]
The domain/default pair of (/Users/btrane/myproject/Pods/Applanga/Applanga.framework/Info, CFBundleShortVersionString) does not exist
Applanga SDK Version:
  File "/tmp/Applanga-Scripts//settingsfile_update.py", line 1
    404: Not Found
       ^
SyntaxError: invalid syntax
```
I also tried setting up a fresh project, added Applanga via Cocoapods, and added this build phase:
```
bash "$SOURCE_ROOT/Pods/Applanga/Applanga.framework/update-settingsfile.sh" "$SOURCE_ROOT/$TARGET_NAME"
```
I'm still getting the same error in the log with the fresh project. I'm currently using Xcode 11.2.1 and Cocoapods 1.8.4.

**Comment by applanga-rich** [comment]
@btrane really sorry but i can't reproduce that here with the same setup, is there any way you could give me more info on your setup? or could you contact me through email then we can setup a quick skype or something so we can solve the issue quickly. 

you can reach me on support@applanga.com

Just mention that you want to talk to Richard and it will get to me.

I'm sure it's something small we can fix quickly

**Comment by btrane** [comment]
@applanga-rich I'm currently running MacOS Catalina 10.15.1. I've narrowed it down to `defaults read` not working on my system with filepaths. The `man defaults` page warns that it won't work with filepaths in the future:
> WARNING: The defaults command will be changed in an upcoming major release to only operate on
               preferences domains. General plist manipulation utilities will be folded into a different
               command-line program.

I was able to modify [the script](https://github.com/applanga/sdk-ios/blob/922b596aa721ad9c23ee9d16b11a81944d81bb06/Applanga.framework/update-settingsfile.sh#L25) using `plutil` instead and submitted a PR (#9) in case it's helpful.

Thanks again for reaching out!
